<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoSimulator</class>
 <widget class="QWidget" name="GeoSimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoSimulator</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QGroupBox" name="InputgroupBox">
         <property name="title">
          <string>Input Data</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QRadioButton" name="radioSingle">
              <property name="text">
               <string>Single Accuracy</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioDouble">
              <property name="text">
               <string>Double Accuracy</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioMemory">
              <property name="text">
               <string>Save Memory</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="basicgroupBox">
            <property name="title">
             <string>Landuse Data</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="laulabel">
                 <property name="text">
                  <string>Path</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="laulineEdit"/>
               </item>
               <item>
                <widget class="QPushButton" name="laubtn">
                 <property name="text">
                  <string> ... </string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="laubtnColor">
                 <property name="minimumSize">
                  <size>
                   <width>131</width>
                   <height>31</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Set NoData Value</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="trangroupBox">
         <property name="title">
          <string>NN Training</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QRadioButton" name="raAve">
              <property name="text">
               <string>Uniform Sampling</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="raSca">
              <property name="text">
               <string>Sampling in Proportion</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="prelabel">
                <property name="text">
                 <string>Percentage of Random Points</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="hidlabel">
                <property name="text">
                 <string>Hidden Layer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="TTlabel">
                <property name="text">
                 <string>Running Times</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QSpinBox" name="PerSpinBox"/>
              </item>
              <item>
               <widget class="QSpinBox" name="HidSpinBox"/>
              </item>
              <item>
               <widget class="QSpinBox" name="TTspinBox"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="savePlabel">
              <property name="text">
               <string>Save Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="savePlineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="savePBtn">
              <property name="text">
               <string> ... </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="divgroupBox">
       <property name="title">
        <string>Driving Data</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QRadioButton" name="divNorrabtnyes">
            <property name="text">
             <string>Normalization</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="divNorrabtnno">
            <property name="text">
             <string>No Nomrmalization</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="divbtn">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="divbtnAdd">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="divbtnSubtract">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="divtableView"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="ANNtrainBtn">
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>NN Training</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="probOutText">
     <property name="minimumSize">
      <size>
       <width>252</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>252</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
